docs/content/releases/4.5.8.md

```diff
@@ -11,7 +11,7 @@ The PostgreSQL Operator is released in conjunction with the [Crunchy Container S
 
 PostgreSQL Operator 4.5.8 release includes the following software versions upgrades:
 
-- [PostgreSQL](https://www.postgresql.org) versions 13.8, 12.12, 11.17, and 10.22 are now available.
+- [PostgreSQL](https://www.postgresql.org) versions 13.9, 12.13, 11.18, and 10.23 are now available.
```

Comment:
:wrench: 
I don't think we want to update these release notes, so I think this change should be reverted.

Reply:
Sure don't.